pos_pricelist/i18n/es.po

```diff
@@ -1,141 +1,179 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-#	* pos_pricelist
+# * point_of_sale
 #
+# Translators:
```

Comment:
Why this large list of translators? They haven't translated the exact terms of this module, so better to remove them.

Reply:
I copied preexisting translations from `point_of_sale` v11, since all terms changed to match those. I just filled the missing ones, so I feel I should keep attribution. 😕 

See https://github.com/odoo/odoo/blob/11.0/addons/point_of_sale/i18n/es.po

Reply:
We are not attributing translation that way and it's going to be deleted any way on next Transifex push, so please remove it.